### ISSUE TITLE
Feature List

### ISSUE THREAD

**Issue by aseelk** [OmerGoldreich/StrollerApp#1]
- **Feature name:** Fun Routes
- Description: A map showing the most interesting walking route between two points
- Requires: Updating client side, developing an algorithm, working with Google Maps API
- Due: Milestone 1
- Status: Client side is ready, algorithm is still developed
- Time Estimation: 1 week
-----------------------------------------------------------------
- **Feature name:** Google/Facebook Login
- Description: Individual users can log in to the application
- Requires: Working with Google/Facebook Developers, updating server
- Due: Milestone 1
- Status: Implemented
- Time Estimation: 1 day
-----------------------------------------------------------------
- **Feature name:** Origin and Destination Search
- Description: Search by the starting and ending point of the route
- Requires: Updating client side, working with Google Maps API
- Due: Milestone 1
- Status: Implemented
- Time Estimation: 1 day
-----------------------------------------------------------------
- **Feature name:** Favorite Routes
- Description: Marking suggested routes as favorites, and loading them to map.
- Requires: saving in database, Developing a mechanism to launch saved routes.
- Due: Milestone 2
- Status: Saving is in process, Database infrastructure is not yet implemented.
- Time Estimation: 7 days
-----------------------------------------------------------------
- **Feature name:** Live Navigation
- Description: Live navigation of the route suggested by the application
- Requires: Working with Google Maps API, supporting client side
- Due: Milestone 2
- Status: Under development
- Time Estimation: Currently unknown- Issues with navigating on a custom route.
-----------------------------------------------------------------
- **Feature name:** Route Ranking
- Description: Providing feedback at the end of the route
- Requires: Supporting client side, saving in database
- Due: Milestone 2
- Status: Under development
- Time Estimation: 1 day
-----------------------------------------------------------------
- **Feature name:** Choose a different route
- Description: User can choose whether he likes the route suggested or he likes to see other options.
- Requires: Supporting client side, calculating multiple routes on server.
- Due: Nice to have
- Status: Pending
- Time Estimation: 3 days.
-----------------------------------------------------------------
- **Feature name:** Irrelevant interests
- Description: After calculation user can filter out fields of interests which led to the chosen route by sweeping them out- and thus he prompts reculculation and possibly a different route will be chosen.
- Requires: Supporting client side, calculating with different parameters on server.
- Due: Nice to have
- Status: Pending
- Time Estimation: 5 days.
-----------------------------------------------------------------

**Comment by aseelk** [comment]
- Feature name: Live Navigation
- Description: Live navigation of the route suggested by the application
- Requires: Working with Google Maps API, supporting client side
- Due: Milestone 2
- Status: Under development
- Time Estimation: 2 days

**Comment by aseelk** [comment]
Feature name: Favorite Routes
Description: Marking suggested routes as favorites
Requires: Supporting client side, saving in database
Due: Milestone 2
Status: Client side is ready, database infrastructure is not ready
Time Estimation: 2 days
